### PR TITLE
Lu 8

### PR DESCRIPTION
**Pull request by sanketXynes**
New Button

### CODE REVIEW

File: src/components/Button/button.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  HTMLButtonElement,
+  ButtonProps
+> = (
+  {
+    type = "fill",
+    icon,
+    label = "BUTTON",
+    color = globalColor.foreground,
+    iconAtEnd = false,
+    bgColor,
```
Comment (xynes-admin) [inline src/components/Button/button.tsx:22]: Please rename it to backgroundColor

File: src/components/Button/button.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    ...props
+  },
+  ref
+) => {
+  const ButtonContainer = styled.button`
+    background-color: ${type !== "label" ? bgColor : "none"};
+    border: ${type !== "outlined" ? "none" : `0.3vw solid ${borderColor} `};
+    padding: calc(${spacing?.padding?.small} / 4) ${spacing?.padding?.small};
+    border-radius: ${borderRadius};
+    flex-direction: ${iconAtEnd && "row-reverse"};
```
Comment (xynes-admin) [inline src/components/Button/button.tsx:34]: Please also add an else statement with "row"

**Review by xynes-admin — CHANGES_REQUESTED** [review]
Please make the following changes and update @sanketXynes

File: src/components/Button/button.tsx (hunk 1 above)
Comment (sanketXynes) [reply src/components/Button/button.tsx]: Updated

File: src/components/Button/button.tsx (hunk 2 above)
Comment (sanketXynes) [reply src/components/Button/button.tsx]: Updated

**Review by archan96 — APPROVED** [review]
Looks Good

**Review by xynes-admin — APPROVED** [review]

**Pull request merged**